src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,57 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+	private final static Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("//(.)\\n(.*)");
```

Comment:
```suggestion
	private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("//(.)\\n(.*)");
```

static을 먼저 작성하셔야 합니다